node/src/components/block_validator.rs

```diff
@@ -188,12 +202,12 @@ where
             }) => {
                 debug!(?block, "validating proposed block");
                 if block.deploy_hashes().count()
-                    > self.chainspec.transaction_config.block_max_deploy_count as usize
+                    > self.chainspec.transaction_config.block_max_standard_count as usize
                 {
                     return responder.respond(false).ignore();
                 }
                 if block.transfer_hashes().count()
-                    > self.chainspec.transaction_config.block_max_native_count as usize
+                    > self.chainspec.transaction_config.block_max_transfer_count as usize
```

Comment:
We should check for `block_max_staking_count` and `block_max_install_upgrade_count` here as well.

Reply:
As per comment above.